daemon/src/maker_inc_connections.rs

```diff
@@ -494,40 +493,39 @@ impl Actor {
 
         use wire::TakerToMaker::*;
         match msg.msg {
-            Protocol { order_id, msg } => match self.setup_actors.get_connected(&order_id) {
```

Comment:
did you mean to remove the `get_connected()` check from here? This doesn't seem like it should be part of the changeset (if anything, you can use `send_async()` inside, after finding the appropriate setup actor, instead of dispatching to all of them).

Reply:
As far as I can tell this `get_connected` check will be implied when we use `send_async` on the `AddressMap` directly. This `get_connected` check here is legacy code for me. We might even be able to *not* expose `get_connected`. I can have a look at that.

Reply:
you are right! I didn't look much into `AddressMap` before, we're looking at the keys whenever we `send` as well, so that part was not needed there beforehand either....
We can remove that method altogether. :)

Reply:
I added another commit that removes the `get_connected` check because it is unused now :)